README.md

```diff
@@ -62,6 +62,14 @@ location.
 
 * Note: you must set each of the below environment variables (AZURE_TENANT_ID, AZURE_CLIENT_ID and AZURE_CLIENT_SECRET) prior to running the script.*
 
+### How to enable logs and retrieve operation logs? 
```

Comment:
Maybe this section should be after "What is this lib/deployer.rb doing?" since it appears to be a more specific section, and could come after explaining the example.